service/src/test/java/bio/terra/cbas/controllers/TestMethodsApiController.java

```diff
@@ -464,6 +468,8 @@ void validDockstoreMethodPostRequest() throws Exception {
     when(cromwellService.describeWorkflow(validRawWorkflow))
         .thenReturn(workflowDescForValidWorkflow);
 
+    when(uuidSource.generateUUID()).thenReturn(UUID.randomUUID()).thenReturn(UUID.randomUUID());
```

Comment:
If this is fine being random, I would imagine we don't need to overwrite the default implementation here?

Reply:
I added this because without this the test was receiving null for `uuidSource`. Is there something else I should be doing?